From 57479f576a540308bc7faf5912a35814ff3e740e Mon Sep 17 00:00:00 2001
From: Arne Coucheron <arco68@gmail.com>
Date: Mon, 3 Jul 2017 01:01:37 +0200
Subject: [PATCH 28/32] Change icon drawable padding

No reason to have this huge gap between icon and text.
Use 4dp like in old Trebuchet.

Change-Id: I0b6e320150778891f05c38be4467e730b568d19b
---
 res/values/dimens.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values/dimens.xml b/res/values/dimens.xml
index efec7717d..4032575ba 100644
--- a/res/values/dimens.xml
+++ b/res/values/dimens.xml
@@ -21,7 +21,7 @@
     <!-- Dynamic Grid -->
     <dimen name="dynamic_grid_edge_margin">8dp</dimen>
     <dimen name="dynamic_grid_page_indicator_line_height">1dp</dimen>
-    <dimen name="dynamic_grid_icon_drawable_padding">8dp</dimen>
+    <dimen name="dynamic_grid_icon_drawable_padding">4dp</dimen>
     <dimen name="dynamic_grid_workspace_top_padding">8dp</dimen>
     <dimen name="dynamic_grid_workspace_page_spacing">8dp</dimen>
     <!-- Minimum space between workspace and hotseat in spring loaded mode -->
-- 
2.17.1

